### ISSUE TITLE
Information button and Anonymous button are not announced as expandable button

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#254]
**Description**
The Information button and anonymous button are not announced as expandable. 
Its current state is also not announced by the screen reader as either "expanded" or "collapsed.

**To Reproduce**
Steps to reproduce the behavior (using screen reader - NVDA):
1. Open the URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Press Alt key to go to the chrome menu and press tab to go to the sidekick extension 
3. Press enter on the sidekick extension
4. Press tab to go to the information button/anonymous and press enter or space key to expand 
5. Notice outuput of screen reader

**Expected behavior**
The information button/anonymous button should have a role of expand/collapse button and should indicate its current state to the user.
This will allow a user to know that pressing this button will reveal content and pressing it again will hide content.

**Screenshots**
Attached is the screenshot
![information button dropdown 7](https://user-images.githubusercontent.com/123964084/219018041-270d0ff4-3ad1-4d5e-a001-766483419bf4.png)

**Additional context**
https://www.w3.org/TR/WCAG21/#name-role-value

**Why it matters**
Providing the appropriate role to a widget enables the user to understand and opreate the widget. In this case, the user cannot tell by the role "button" that pressing this button will reveal new content on the page. Using an expand/collapse widget or similar is more appropriate.

**Comment by rofe** [comment]
@puja-iaccessible is it sufficient to add `aria-expanded` attributes (true or false) to the dropdowns?

**Comment by manish10** [comment]
Yes, Please look at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded on how to correctly add the aria-expanded tag to the button.
